test/e2e/framework/framework.go

```diff
@@ -333,7 +333,7 @@ func (f *Framework) AfterEach() {
 		if len(nsDeletionErrors) != 0 {
 			messages := []string{}
 			for namespaceKey, namespaceErr := range nsDeletionErrors {
-				messages = append(messages, fmt.Sprintf("Couldn't delete ns: %q: %s", namespaceKey, namespaceErr))
+				messages = append(messages, fmt.Sprintf("Couldn't delete ns: %q: %s (%#v)", namespaceKey, namespaceErr, namespaceErr))
```

Comment:
That doesn't seem to help here: https://github.com/kubernetes/kubernetes/issues/32045#issue-174946883, since the error is an aggregate :/